samples/net/sockets/civetweb/README.rst

```diff
@@ -0,0 +1,38 @@
+.. _sockets-civetweb-sample:
+
+Civetweb sample
+###############
+
+Overview
+********
+
+This sample is a demonstrator of the HTTP API provided by the Civetweb module.
+It makes Zephyr act as a HTTP server and demonstrates selected features of the Civetweb API.
+
+The source code for this sample application can be found at:
+:zephyr_file:`samples/net/sockets/civetweb`.
+
+Requirements
+************
+
+- A board with hardware networking
+
+Building and Running
+********************
+
+This sample was tested on the Atmel SAM E70 Xplained board, so this is the recommended target.
+
+Build it with:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/sockets/civetweb
+   :board: sam_e70_xplained
+   :goals: build
+   :compact:
+
+The sample application uses a static IP configuration.
+
+After flashing the board, the sever can be accessed with the web browser of your choice at ``10.0.0.111:8080``.
+
+The IP configuration can be changed in Zephyr config.
+The default port can be changed in the sources of the sample.
```

Comment:
You should mention that this sample is not a general HTTP server accessing a filesystem when receiving http requests.  Instead only three "uris" are supported:  ``/``, ``/info``, and ``/history``, and explain the uri responses and what happens if any non-supported uris are received.

Reply:
Updated